libs/eavmlib/src/uart.erl

```diff
@@ -19,10 +19,13 @@
 %
 
 -module(uart).
--export([open/2, close/1, read/1, write/2]).
+-export([open/1, open/2, close/1, read/1, write/2]).
 
 open(Name, Opts) ->
-    open_port({spawn, "uart"}, [{name, Name} | Opts]).
+    open_port({spawn, "uart"}, [{peripheral, Name} | Opts]).
```

Comment:
Should `open/2` also use `migrate_config(Opts)`? It seems more likely that someone using old configuration parameters would not be using the new `open/1` function.

Reply:
Fixed.